packages/mobile/src/exchange/ExchangeConfirmationCard.tsx

```diff
@@ -34,16 +34,18 @@ export default function ExchangeConfirmationCard(props: Props) {
   const totalFee = new BigNumber(tobinTax).plus(fee)
   const feeCurrency = Currency.Dollar
 
-  const localAmount = (isSellGoldTx ? makerAmount : takerAmount).localAmount!
```

Comment:
This would cause a crash when tapping the exchange feed item when no local amount is set.